scripts/spack_packages/umpire/package.py

```diff
@@ -189,20 +189,26 @@ def initconfig_hardware_entries(self):
             rocm_root = hip_root + "/.."
             entries.append(cmake_cache_path("HIP_ROOT_DIR",
                                         hip_root))
-            entries.append(cmake_cache_path("HIP_CLANG_PATH",
+            entries.append(cmake_cache_path("ROCM_ROOT_DIR",
+                                        rocm_root))
+            entries.append(cmake_cache_path("HIP_PATH",
                                         rocm_root + '/llvm/bin'))
+            #entries.append(cmake_cache_path("HIP_CLANG_PATH",
+            #                            rocm_root + '/llvm/bin'))
             entries.append(cmake_cache_string("HIP_HIPCC_FLAGS",
                                         '--amdgpu-target=gfx906'))
-            entries.append(cmake_cache_string("HIP_RUNTIME_INCLUDE_DIRS",
-                                        "{0}/include;{0}/../hsa/include".format(hip_root)))
-            hip_link_flags = "-Wl,--disable-new-dtags -L{0}/lib -L{0}/../lib64 -L{0}/../lib -Wl,-rpath,{0}/lib:{0}/../lib:{0}/../lib64 -lamdhip64 -lhsakmt -lhsa-runtime64".format(hip_root)
+            #entries.append(cmake_cache_string("HIP_RUNTIME_INCLUDE_DIRS",
+            #                            "{0}/include;{0}/../hsa/include".format(hip_root)))
+            #hip_link_flags = "-Wl,--disable-new-dtags -L{0}/lib -L{0}/../lib64 -L{0}/../lib -Wl,-rpath,{0}/lib:{0}/../lib:{0}/../lib64 -lamdhip64 -lhsakmt -lhsa-runtime64".format(hip_root)
+            entries.append(cmake_cache_option("UMPIRE_ENABLE_TOOLS", False))
+            hip_link_flags = ""
             if '%gcc' in spec:
                 gcc_bin = os.path.dirname(self.compiler.cxx)
                 gcc_prefix = join_path(gcc_bin, '..')
                 entries.append(cmake_cache_string("HIP_CLANG_FLAGS", "--gcc-toolchain={0}".format(gcc_prefix))) 
                 entries.append(cmake_cache_string("CMAKE_EXE_LINKER_FLAGS", hip_link_flags + " -Wl,-rpath {}/lib64".format(gcc_prefix)))
             else:
-                entries.append(cmake_cache_string("CMAKE_EXE_LINKER_FLAGS", hip_link_flags))
+                entries.append(cmake_cache_string("CMAKE_EXE_LINKER_FLAGS", "-Wl,-rpath={0}/llvm/lib/".format(rocm_root)))
```

Comment:
Should we await feedback from LC before making this particular change?

Reply:
Indeed we should!

Reply:
Documenting what we talked about in dev meeting: We will keep this change for now and if LC develops a better fix later on, then we'll go from there.